package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@divriots/starter-lit-lion",
```

Comment:
It can be setup to use the @divriots scope.
Not a big deal for now of course

Reply:
Oh hm weird I did not actually change this consciously, not sure how or why it was changed while working in backlight.dev

Reply:
Ahahaha sorry I didn't know but the product does that automatically :)

Forget about my comment!